src/libs/Navigation/AppNavigator/createModalCardStyleInterpolator.ts

```diff
@@ -0,0 +1,29 @@
+import type {StackCardInterpolatedStyle, StackCardInterpolationProps} from '@react-navigation/stack';
+import {Animated} from 'react-native';
+import {StyleUtilsType} from '@styles/utils';
+import variables from '@styles/variables';
+
+export default (StyleUtils: StyleUtilsType) =>
```

Comment:
Could give this function a name instead of exporting an anonymous function. Better for debug?